### DISCUSSION TITLE
how to use the -se argument

### DISCUSSION THREAD

**Discussion by aSocialMenace** [nerdyrodent/VQGAN-CLIP#110] [Q&A]
ok so I finally have this up and working! and I love the fact that im running this locally however when you use this in google colab it creates an image for every 50th iteration. I tried feeding it a command using the -se arguments but it just overwrites the same output file with ever 50th iteration (as i used -se 50) I would really like to save ever 50th iteration separately. is there a way to do this?? is there a way of passing something with the -o argument that I am missing?? any help would be greatly appreciated!!

**Comment by jonnm4** [comment]
-se only updates your "working image" you still have to copy it somewhere else and rename it. Or, yes. It just get rewritten. Personally, I use -i 50; then copy the file some where, then -ii "Working_Img.png" -i 50 to start up again from where you left off in theory you could make a multi-threaded program to run the python generate.py AND deal with the files. But, I don't know how.
